<launch>

    <node name="marker_spawner" pkg="cartesian_interface" type="marker_spawner" output = "screen" /> 
    
    <node type="rviz"
            name="rviz"
            pkg="rviz"
            args="-d $(find awesome_leg)/rviz/cartesio_imp_cntrl_marker.rviz"/>
            output="screen"/>

</launch>
